docs/docs/guides/guidelines.md

```diff
@@ -0,0 +1,155 @@
+
+### Do:
+
+- Ask questions that reflect real-life situations and needs.
+- Ask questions that might be directed towards search engines or specialists.
+- Make requests that encourage lateral thinking and require specialized knowledge.
+- Use a mix between questions that are straightforward and questions without a clear answer.
+- Introduce a variety in prompts by using different phrasing, degrees of politeness or amount of context given.
+- Consider the previous replies and prompts that lead up to the current one.
+- Try to build upon the topic and ask a sensible follow-up question when replying to the assistant.
+
+### Don't:
+
+- Write prompts without a clear request.
+- Make requests that override the original purpose of the assistant, ie. jailbreak the model.
+- Use the same prompt repeatedly.
+- Change the topic of a conversation without prefacing it accordingly when replying to the assistant.
+- Leave typos and grammatical errors in the prompt.
+- Reply in a language different from the one intended for the data set.
+
+## 4. Classifying an assistant reply
+
+### Do:
+
+- Rate every criteria of each reply, unless it can't be discerned because it is spam or inappropriate.
+- Judge quality based on how well the reply adheres to the guidelines. Factual accuracy and helpfulness are first and foremost.
+- Make sure to read the reply thoroughly.
+- Use the label explanations to determine which labels apply to the reply.
+- Research to make sure whether the reply is factually accurate.
+- Skip if you are unable to determine the validity of reply.
+
+### Don't:
+
+- Judge quality based on personal beliefs. Assuming an opinion was warranted, fulfills the users request and doesn't violate any guidelines, it should not impact the rating of the reply.
+- Skip a label just because the reply is spam. Each label can help the model improve.
+- Rate a reply's ability to answer a question correctly if you aren't certain.
+
+## 5. Classifying an initial prompt or user reply
+
+### Do:
+
+- Rate every criteria of each prompt, unless it can't be discerned because it is spam or inappropriate.
+- Judge quality based on how well the prompt adheres to the guidelines.
+- Make sure to read the prompt thoroughly.
+- Use the label explanations to determine which labels apply to the prompt.
+
+### Don't:
+
+- Judge quality based on personal beliefs. The opinion of the user should not impact the rating of the prompt.
+- Skip a label just because the reply is spam. Each label can help the model improve.
+
+## 6. Ranking assistant replies
+
+### Do:
+
+- Make sure to read every available reply.
+- Think about which reply best satisfies the request of the user.
+- Rank replies based on how well they adhere to the guidelines. Factual accuracy and helpfulness are first and foremost.
+- Penalize replies that fail to provide adequate warnings or caveats.
+- Penalize replies that are difficult to read due to a lack of formatting, capitalization or other errors.
```

Comment:
Maybe add:

Penalize vague and empty language if the goal is to provide information. Unnecessary padding just dilutes the useful information that is already in the reply. Everything else being equal a more concise answer is preferred.